vendor/assets/javascripts/gridstack.js

```diff
@@ -0,0 +1,1738 @@
+/**
```

Comment:
Is this library (and the other 3rd party stuff) avaliable as gems/npm that you can reference, rather than include? 

This will make staying up to date much easier.